doc/DOCUMENTATION.org

```diff
@@ -2106,6 +2106,15 @@ Windows manipulation commands (start with ~w~):
 | ~SPC w W~              | select window using [[https://github.com/abo-abo/ace-window][ace-window]]                                              |
 | ~SPC w x~              | delete a window and its current buffer (does not delete the file)           |
 
+You can use a prefix argument with some keybindings to make them more potent:
```

Comment:
What does potent mean exactly ? What additional power do they get ? :-)

Reply:
I king of ignored the documentation until the last moment. Will rewrite this.